try:
    print("hello world")
    printss("123")
except Exception as e:
    print(e)